authorization/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AuthorizationHandler.java

```diff
@@ -70,6 +70,8 @@ public APIGatewayProxyResponseEvent handleRequest(
             authorizationValidatorService.validate(authenticationRequest, sessionItem);
 
             // create authorization
+            sessionService.createAuthorizationCode(sessionItem);
+
```

Comment:
Note: calling the code to create authorization code. This is now part of the Authorization Handler. It still exists in the cris as well.